src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -1378,6 +1378,23 @@ public async Task UpdateUserGroupsAsync(OrganizationUser organizationUser, IEnum
             await _eventService.LogOrganizationUserEventAsync(organizationUser,
                 EventType.OrganizationUser_UpdatedGroups);
         }
+        
+        public async Task UpdateUserResetPasswordEnrollmentAsync(Guid organizationId, Guid organizationUserId, string resetPasswordKey)
+        {
+            var orgUser = await _organizationUserRepository.GetByOrganizationAsync(organizationId, organizationUserId);
```

Comment:
We need to confirm that this orgUser.UserId == the calling user's userid

Reply:
Added 👍